config/config.toml

```diff
@@ -3,11 +3,29 @@
 #   ~/.config/dust/config.toml
 #   ~/.dust.toml
 
+# Print tree upside down (biggest highest)
 reverse=true
+
+# Subdirectories will not have their path shortened
 display-full-paths=true
+
+# Use file length instead of blocks
 display-apparent-size=true
+
+# No colors will be printed
 no-colors=true
+
+# No percent bars or percentages will be displayed
 no-bars=true
+
+# No total row will be displayed
 skip-total=true
+
+# Do not display hidden files
 ignore-hidden=true
+
+# print sizes in powers of 1000 (e.g., 1.1G)
 iso=true
+
+# Depth to show
+depth=1
```

Comment:
I might remove this from the default config later - but you will be able to add it back in to your local config if you want it.

Reply:
Oh that's the default config? I thought it's just an example with a few options.

Reply:
Removed it from this file.

Reply:
actually this was my bad it was examples - sorry.